lib/ansible/galaxy/dependency_resolution/dataclasses.py

```diff
@@ -106,24 +116,24 @@ def _validate_v1_source_info_schema(namespace, name, version, provided_arguments
     return validation_result.error_messages
 
 
-def _is_collection_src_dir(dir_path):
+def _is_collection_src_dir(dir_path: object) -> bool:
```

Comment:
Why are paths like this annotated with `object`?

Reply:
Fixed locally. Marking as resolved.